### ISSUE TITLE
Consuming a lot of memory when calculating segmentation AUROC

### ISSUE THREAD

**Issue by gaowq2017** [gudovskiy/cflow-ad#15]
Hi, thank you for sharing the code. Consuming a lot of memory when calculating segmentation AUROC as follow:
![捕获](https://user-images.githubusercontent.com/29697367/140691039-56727cdc-a60e-40de-8e8c-7b1210e950f7.PNG)
 When the code run to it as follow:
![image](https://user-images.githubusercontent.com/29697367/140691160-122e7f1b-c778-4604-aaf5-bd7a8afd1486.png)

**Comment by gudovskiy** [comment]
@gaowq2017 yes it is normal. This is an external function to get precision metrics
